tests/testsuite/deduplicate_workspace.rs

```diff
@@ -0,0 +1,861 @@
+//! Tests for deduplicating Cargo.toml fields with { workspace = true }
```

Comment:
nit: Should we update this comment and filename to refer to "inheritable" rather than "de-duplicate" as that is how we are referring to it everywhere else in this PR (documentation, feature flag, types, etc)

Reply:
do you like `//! Tests for inheriting Cargo.toml fields with { workspace = true }` and `inheritable_workspace_fields.rs` better?

Reply:
That works.